app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -46,7 +46,7 @@ public Account(String name, String uid)
     /* ********* Getters & Setters: ********* */
 
     public static Account getCurrentAccount() {
-        return currentAccount;
+        return Account.currentAccount;
```

Comment:
I think that these two lines are doing the same

Reply:
youre right, i changed it because i thought this is a problem but it doesnt matter whats written here